force-app/main/default/lwc/changeAmountOrFrequencyModal/changeAmountOrFrequencyModal.js-meta.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>54.0</apiVersion>
+    <isExposed>true</isExposed>
+    <targets> 
+        <target>lightning__AppPage</target>
+        <target>lightning__RecordPage</target>
+        <target>lightning__HomePage</target>
```

Comment:
Are all of these necessary? Once we release this, the `targets` cannot be changed, and having a component show up in an area where it doesn't belong can be confusing (this comment applies to the other components as well)

Reply:
you are right, these components should not be exposed since they are modal related only to the actions of this component